### PR TITLE
Mention more third-party exporters and integrations.

### PR DESCRIPTION
**Pull request by juliusv**

### CODE REVIEW

File: content/docs/instrumenting/exporters.md [hunk 1]
```diff
@@ -25,3 +26,23 @@ following is a list of existing third-party exporters:
 The [JMX exporter](https://github.com/prometheus/jmx_exporter) can export from a
 wide variety of JVM-based applications, for example [Kafka](http://kafka.apache.org/) and
 [Cassandra](http://cassandra.apache.org/).
+
+Unofficial third-party exporters:
+
```
Comment (brian-brazil) [inline content/docs/instrumenting/exporters.md]: Can we add a disclaimer that these may not follow best practices or something?

Comment (juliusv) [inline content/docs/instrumenting/exporters.md]: I added something to that effect, and changed the text/headline structure a bit.

**Comment by juliusv** [comment]
Added https://atmospherejs.com/sevki/prometheus-exporter

**Comment by juliusv** [comment]
Merging this if no objections in 3... 2... 1...

**Comment by fabxc** [comment]
:+1:

**Pull request merged**